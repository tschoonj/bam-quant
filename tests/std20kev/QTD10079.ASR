File: C:\xrfdaten\2014KW30\chris\std20kev\std10079.spe
Start: 21.07.2014 18:34:31
Stop: 21.07.2014 18:36:11
BESSY: 298.443504 298.0635205
IONI: 3.048193E-11 3.091714E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 83195
Run: 100000
Dead: 17
ICR: 52090
OCR: 0
Accumulated BESSY: 59662.1745697
Accumulated Ioni: 6.14670765E-09
N Accumulation: 200
Normfactor:  3.9110216e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Au
exp_x 111.260398864746 111.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 134.9999
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  79   2      9.707   1835775.      1268.     209.63
