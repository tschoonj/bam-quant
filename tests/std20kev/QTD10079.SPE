$DATA:
0 2000
19261
12518
5410
1639
361
60
16
11
4
3
1
1
2
1
0
4
154
1029
1233
1203
1199
1136
1083
1043
936
904
889
830
788
786
809
746
741
718
626
734
635
679
603
594
582
579
590
577
542
526
527
545
514
500
480
466
499
486
471
455
489
449
470
476
463
447
397
387
423
420
412
447
413
435
445
415
435
406
451
424
406
363
385
423
414
416
407
433
393
432
420
403
389
355
394
412
419
361
379
410
428
427
469
576
696
918
1407
2037
2883
3409
4009
4106
4007
3694
3152
2361
1777
1151
748
558
448
423
461
457
469
501
434
429
352
362
355
333
350
326
360
356
337
345
353
402
373
391
396
426
382
390
344
371
392
390
395
408
389
341
366
310
291
311
301
309
329
308
325
310
300
266
300
291
307
283
325
273
292
311
296
302
302
310
305
239
257
265
302
271
280
302
261
278
292
268
284
263
291
323
265
290
279
281
302
287
268
305
280
285
297
258
277
277
288
261
278
271
291
278
268
264
259
282
238
271
262
286
272
301
289
290
287
283
273
292
231
280
268
246
284
276
240
274
281
267
279
273
282
278
243
267
273
263
274
266
280
258
296
271
261
261
276
272
265
277
257
275
304
263
302
273
253
285
255
253
295
281
274
284
266
278
268
300
255
267
272
246
288
281
280
255
261
261
255
270
293
299
298
268
263
304
306
306
265
272
293
278
285
304
281
309
277
286
314
290
277
267
307
293
277
296
309
342
351
368
359
364
342
366
335
297
342
279
287
292
308
307
300
301
294
316
286
334
325
350
324
322
299
321
315
298
278
284
269
304
323
341
287
313
337
317
290
302
293
303
337
303
310
359
293
325
307
315
383
328
334
326
332
353
358
348
325
311
375
367
328
326
322
386
372
359
338
379
380
386
391
450
446
500
536
574
583
610
630
660
580
589
547
533
492
517
433
495
495
477
520
513
521
605
694
802
1007
1409
1989
2847
3934
5248
6352
7486
8229
8176
7827
6820
5642
4225
3142
2259
1594
1162
931
792
769
720
733
694
623
697
694
655
618
635
655
668
679
655
698
764
709
747
822
824
863
927
924
1026
1062
1016
1022
1107
1153
1194
1271
1375
1459
1621
2024
2501
3512
4896
7934
12358
19226
30171
44634
64585
89152
116098
143230
165690
180500
183339
172563
152856
125602
96253
68321
45888
28766
17305
9974
5623
3461
2248
1564
1183
1100
998
895
882
897
984
1091
1301
1632
2068
2557
2954
3501
3788
4107
4041
3648
3224
2682
2241
1768
1382
1105
918
809
726
715
685
725
710
684
695
755
721
782
793
826
842
846
897
942
934
1038
1033
1182
1303
1418
1661
1986
2425
3210
4341
5611
7330
9301
11377
13428
15073
16252
17128
18226
19959
23540
29958
40310
55032
72653
93027
112607
127897
138176
142148
137742
129098
118175
105675
95221
84114
73986
64211
54826
45151
35749
27259
20084
14121
10090
7231
5546
4585
3966
4106
4145
4094
4028
3888
3411
3062
2693
2245
1905
1506
1287
1111
975
806
726
643
533
413
351
295
243
210
163
177
183
158
129
132
178
153
153
150
153
174
171
176
166
181
180
174
157
202
168
185
210
235
281
305
357
465
568
603
725
728
798
806
745
754
659
587
522
551
687
806
1117
1647
2371
3539
5155
7017
9421
11583
13403
15013
15838
15618
14512
13072
11035
8864
6800
5012
3683
2875
2387
2402
2563
2815
3375
3830
4450
4444
4792
4548
4425
4009
3536
3006
2467
1971
1449
1082
845
614
398
297
249
198
196
185
158
188
220
257
345
405
482
547
589
617
594
618
496
432
355
265
204
148
108
71
36
46
25
15
21
17
18
12
13
8
8
17
9
18
6
9
7
8
9
9
7
4
5
14
10
9
13
10
11
9
11
5
4
10
9
5
7
11
10
10
6
14
7
4
7
4
9
11
4
4
4
11
12
17
11
20
11
29
18
39
28
30
33
35
25
29
23
17
16
19
7
4
6
10
10
12
7
10
8
3
8
5
7
3
7
12
5
12
6
6
6
7
4
5
9
9
4
6
2
4
2
3
6
4
2
4
4
6
9
5
8
4
5
5
5
6
3
11
6
1
4
9
2
4
2
4
5
12
4
8
5
7
4
4
4
7
5
10
6
5
4
7
4
6
7
11
7
6
6
3
5
5
11
14
10
7
7
14
10
6
8
9
8
6
14
5
6
8
8
5
4
9
9
15
14
13
13
16
16
14
23
29
25
30
26
35
27
19
24
15
14
8
11
10
5
15
9
6
9
12
6
6
7
12
5
7
9
10
12
6
9
8
9
7
8
10
9
15
13
14
12
21
17
11
14
18
19
19
27
35
46
52
60
92
97
137
142
154
220
257
236
259
272
295
278
257
231
254
168
147
120
98
84
95
68
98
73
54
78
74
77
78
99
137
135
146
165
187
194
212
193
223
207
212
188
212
158
177
149
137
125
128
109
69
71
58
44
51
26
23
21
18
19
13
16
4
12
5
11
10
9
8
14
12
21
13
20
13
18
28
45
54
54
49
59
70
91
106
118
116
175
206
228
290
297
367
410
403
414
490
442
404
364
374
343
299
276
233
245
200
176
131
89
85
72
63
41
41
36
34
35
21
28
17
15
14
13
31
14
15
14
18
16
16
17
18
13
9
12
6
12
11
11
7
6
9
1
3
9
7
4
2
7
7
8
8
9
6
6
11
5
12
23
20
22
21
34
29
32
42
37
59
47
67
83
81
102
103
119
140
152
174
182
192
210
211
217
241
231
224
224
221
202
185
192
159
167
137
145
115
96
77
65
59
46
47
50
33
37
30
26
25
29
17
36
27
15
26
17
17
15
13
6
8
8
6
2
5
6
2
7
5
3
2
5
4
1
4
2
5
3
3
5
1
1
0
0
2
2
3
3
0
3
2
3
1
3
4
3
7
6
7
6
10
11
11
12
5
15
11
14
19
19
26
21
19
29
33
47
44
41
40
51
35
30
44
33
38
32
32
22
20
25
21
12
19
12
19
11
17
11
10
13
9
9
17
9
8
8
6
10
7
2
6
1
2
3
2
0
1
5
4
2
2
3
3
3
5
1
3
4
0
1
1
0
1
0
4
1
0
1
1
0
1
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
2
0
0
0
1
0
0
0
3
3
1
3
3
1
1
2
3
5
1
2
4
5
3
1
1
2
3
0
3
1
1
2
3
2
2
0
3
0
0
2
3
3
0
0
2
1
2
1
1
0
0
0
1
1
1
2
0
2
0
0
1
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
2
0
0
0
1
0
0
0
0
0
0
0
0
0
3
1
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
1
0
0
2
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
2
1
0
1
3
0
1
0
1
0
0
1
2
1
0
2
1
0
1
0
0
1
1
1
0
1
0
0
0
1
0
1
1
1
1
0
3
0
0
2
1
0
2
0
1
0
1
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
1
0
2
0
0
0
0
1
0
0
1
0
0
1
0
0
0
0
1
0
0
0
0
2
1
0
1
0
1
1
2
0
2
0
1
0
0
0
1
0
0
1
1
0
0
1
1
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
1
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10079.spe
Start: 21.07.2014 18:34:31
Stop: 21.07.2014 18:36:11
BESSY: 298.443504 298.0635205
IONI: 3.048193E-11 3.091714E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 83195
Run: 100000
Dead: 17
ICR: 52090
OCR: 0
Accumulated BESSY: 59662.1745697
Accumulated Ioni: 6.14670765E-09
N Accumulation: 200
Normfactor:  3.9110216e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Au
 
exp_x 111.260398864746 111.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 134.9999
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  79   2      9.707   1835775.      1268.     209.63
