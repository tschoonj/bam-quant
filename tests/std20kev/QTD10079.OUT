     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:57  บ   
     บ  Spectrum: QTD10079.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =   145.5   last change =   .00%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -25.1 ๑   100.0   บ     -25.1 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.211 ๑   2.000   บ    20.211 ๑    .001   บ   
     บ det NOISE (eV) บ       82.5 ๑    40.0   บ      82.5 ๑     2.4   บ   
     บ  FANO factor   บ       .131 ๑    .050   บ      .131 ๑    .001   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Au-L1                            291973. ๑    836.           บ   
     บ    L1M3       11.610    .39347      114883. ๑    329.   1269.2  บ   
     บ              575.681    197.90         242.           9.85E-01  บ   
     บ    L1M2       11.205    .33922       99043. ๑    284.    481.6  บ   
     บ              555.642    195.03         256.           9.83E-01  บ   
     บ    L1N3       13.809    .11093       32388. ๑     94.    762.5  บ   
     บ              684.483    212.82         175.           9.86E-01  บ   
     บ    L1N2       13.710    .08795       25679. ๑     75.    877.8  บ   
     บ              679.585    212.17         178.           9.88E-01  บ   
     บ    L1O3       14.291    .03558       10388. ๑     33.     94.8  บ   
     บ              708.331    215.96         180.           9.87E-01  บ   
     บ    L1M5       12.147    .01707        4983. ๑     21.     11.6  บ   
     บ              602.250    201.65         224.           9.83E-01  บ   
     บ    L1M4       12.062    .01137        3321. ๑     18.     10.6  บ   
     บ              598.045    201.06         227.           9.83E-01  บ   
     บ    L1N5       14.020    .00270         790. ๑     14.    366.3  บ   
     บ              694.923    214.20         188.           9.86E-01  บ   
     บ    L1N4       13.999    .00170         497. ๑     13.    404.4  บ   
     บ              693.884    214.06         170.           9.86E-01  บ   
     บ                                                                 บ   
     บ  2 Au-L2                           1491523. ๑   1411.           บ   
     บ    L2M4       11.442    .78916     1177053. ๑   1114.    914.2  บ   
     บ              567.368    196.71         248.           9.84E-01  บ   
     บ    L2N4       13.382    .16464      245567. ๑    233.   2058.1  บ   
     บ              663.356    210.01         187.           9.87E-01  บ   
     บ    L2M1       10.308    .02147       32028. ๑     34.    246.1  บ   
     บ              511.260    188.51         257.           9.79E-01  บ   
     บ    L2N1       12.974    .00561        8360. ๑     16.     19.7  บ   
     บ              643.169    207.28         199.           9.86E-01  บ   
     บ    L2O4       13.730    .01722       25679. ๑     28.    836.7  บ   
     บ              680.574    212.30         177.           9.86E-01  บ   
     บ    L2O1       13.626    .00110        1645. ๑     14.   1137.8  บ   
     บ              675.428    211.62         180.           9.87E-01  บ   
     บ    L2M3       10.992    .00080        1190. ๑     16.    845.5  บ   
     บ              545.103    193.50         264.           9.82E-01  บ   
     บ                                                                 บ   
     บ  3 Au-L3                           2355215. ๑   1627.           บ   
     บ    L3M5        9.716    .69991     1648430. ๑   1139.    209.6  บ   
     บ              481.969    184.08         278.           9.75E-01  บ   
     บ    L3M4        9.628    .07954      187345. ๑    130.    162.5  บ   
     บ              477.615    183.41         281.           9.74E-01  บ   
     บ    L3N5       11.585    .13890      327139. ๑    227.   1309.0  บ   
     บ              574.444    197.72         243.           9.85E-01  บ   
     บ    L3N4       11.567    .01543       36346. ๑     30.   1308.9  บ   
     บ              573.553    197.60         244.           9.85E-01  บ   
     บ    L3M1        8.494    .03888       91564. ๑     66.    184.0  บ   
     บ              421.507    174.58         324.           9.63E-01  บ   
     บ    L3N1       11.160    .00977       23006. ๑     23.    560.0  บ   
     บ              553.415    194.70         258.           9.83E-01  บ   
     บ    L3O1       11.811    .00192        4518. ๑     16.    622.1  บ   
     บ              585.626    199.31         236.           9.86E-01  บ   
     บ    L3O5       11.916    .01565       36867. ๑     30.    251.8  บ   
     บ              590.821    200.04         232.           9.86E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                           19455. ๑    141.           บ   
     บ    L3O5       21.149    .25239        4975. ๑     37.      4.1  บ   
     บ             1047.653    256.42          49.           9.26E-01  บ   
     บ               19.414    .19674        3878. ๑     29.      4.1  บ   
     บ              961.810    246.81          75.           9.61E-01  บ   
     บ    L2N1       21.297    .10252        2021. ๑     16.      2.6  บ   
     บ             1054.958    257.23          47.           9.22E-01  บ   
     บ    L3M5       22.884    .08095        1595. ๑     13.      5.4  บ   
     บ             1133.497    265.69          32.           8.76E-01  บ   
     บ    L3N5       23.032    .06576        1296. ๑     11.      3.6  บ   
     บ             1140.802    266.46          31.           8.72E-01  บ   
     บ    L3M1       23.089    .05266        1038. ๑      9.      2.4  บ   
     บ             1143.641    266.76          30.           8.70E-01  บ   
     บ    L3N1       24.824    .03378         666. ๑      6.      5.3  บ   
     บ             1229.485    275.68          16.           8.11E-01  บ   
     บ               20.904    .02615         515. ๑      8.      5.9  บ   
     บ             1035.507    255.09          52.           9.32E-01  บ   
     บ               18.201    .01963         387. ๑     10.      2.0  บ   
     บ              901.792    239.87          97.           9.77E-01  บ   
     บ               23.459    .01830         361. ๑      6.      3.2  บ   
     บ             1161.945    268.69          27.           8.58E-01  บ   
     บ               22.639    .01678         331. ๑      6.      7.1  บ   
     บ             1121.351    264.40          35.           8.84E-01  บ   
     บ               24.972    .01372         270. ๑      4.      7.7  บ   
     บ             1236.789    276.42          15.           8.05E-01  บ   
     บ               23.179    .01336         263. ๑      6.      1.7  บ   
     บ             1148.106    267.23          29.           8.67E-01  บ   
     บ               19.936    .01259         248. ๑      8.    119.8  บ   
     บ              987.635    249.74          66.           9.52E-01  บ   
     บ               25.194    .01174         231. ๑      4.      7.9  บ   
     บ             1247.788    277.54          13.           7.98E-01  บ   
     บ               21.623    .00790         156. ๑      7.      2.2  บ   
     บ             1071.106    258.99          43.           9.13E-01  บ   
     บ               20.015    .00687         135. ๑      8.    130.1  บ   
     บ              991.545    250.19          65.           9.51E-01  บ   
     บ               22.786    .00681         134. ๑      6.      5.9  บ   
     บ             1128.655    265.17          33.           8.79E-01  บ   
     บ               20.084    .00511         101. ๑      8.    126.6  บ   
     บ              994.940    250.57          64.           9.49E-01  บ   
     บ               23.358    .00507         100. ๑      5.      3.4  บ   
     บ             1156.950    268.16          28.           8.61E-01  บ   
     บ               25.342    .00477          94. ๑      4.      7.0  บ   
     บ             1255.093    278.28          12.           7.92E-01  บ   
     บ               21.750    .00441          87. ๑      6.      1.9  บ   
     บ             1077.389    259.67          41.           9.10E-01  บ   
     บ               26.764    .00352          69. ๑      2.      3.0  บ   
     บ             1325.472    285.32           6.           7.41E-01  บ   
     บ               24.579    .00350          69. ๑      4.      1.8  บ   
     บ             1217.338    274.43          17.           8.19E-01  บ   
     บ               21.876    .00263          52. ๑      6.      1.4  บ   
     บ             1083.623    260.35          40.           9.06E-01  บ   
     บ               27.134    .00245          48. ๑      2.      2.8  บ   
     บ             1343.776    287.12           5.           7.28E-01  บ   
     บ               23.998    .00223          44. ๑      5.      2.0  บ   
     บ             1188.617    271.47          22.           8.40E-01  บ   
     บ               23.506    .00206          41. ๑      5.      3.1  บ   
     บ             1164.254    268.93          26.           8.56E-01  บ   
     บ               22.681    .00179          35. ๑      6.      7.3  บ   
     บ             1123.454    264.62          34.           8.83E-01  บ   
     บ               21.898    .00179          35. ๑      6.      1.6  บ   
     บ             1084.693    260.46          40.           9.06E-01  บ   
     บ               21.820    .00178          35. ๑      6.      1.6  บ   
     บ             1080.854    260.05          41.           9.08E-01  บ   
     บ               25.733    .00143          28. ๑      3.      1.4  บ   
     บ             1274.460    280.23          10.           7.78E-01  บ   
     บ               19.691    .00130          26. ๑      8.     44.8  บ   
     บ              975.489    248.37          70.           9.57E-01  บ   
     บ               24.948    .00122          24. ๑      4.      7.1  บ   
     บ             1235.642    276.30          15.           8.06E-01  บ   
     บ               24.416    .00115          23. ๑      4.      1.0  บ   
     บ             1209.297    273.61          19.           8.25E-01  บ   
     บ               23.555    .00114          22. ๑      5.      3.2  บ   
     บ             1166.697    269.18          26.           8.54E-01  บ   
     บ               25.298    .00106          21. ๑      4.      7.5  บ   
     บ             1252.937    278.06          13.           7.94E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
